src/svgs/glyphs/acne-body-spray-glyph.svg

```diff
@@ -1,7 +1,7 @@
 <svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
   <title>Acne Body Spray Glyph</title>
-  <path d="M28.688 8L26.3874 10.3573L27.267 11.2587L29.5676 8.90133L28.688 8Z" fill="currentColor"/>
-  <path d="M34 14.7023H28.6411V15.879H34V14.7023Z" fill="currentColor"/>
-  <path d="M27.24 19.4555L26.3603 20.3567L28.6606 22.7143L29.5403 21.8131L27.24 19.4555Z" fill="currentColor"/>
-  <path d="M25.7703 17.2517V13.6238H22.9952V11.6627H16.0096V18.6245H14V40H25.0048V18.6245H22.9952V17.2517H25.7703ZM22.9952 14.8004H24.622V16.0751H22.9952V14.8004ZM23.8565 38.8234H15.244V19.8991H23.8565V38.8234ZM17.1579 12.9374H21.8469V18.7225H17.1579V12.9374Z" fill="currentColor"/>
+  <path d="M23.51 16.32H24.08C24.8 16.32 25.39 15.73 25.39 15.01V14.49C25.39 13.77 24.8 13.18 24.08 13.18H23.51C23.51 13.18 23.51 12.68 23.51 12.6C23.51 11.44 21.9 10.91 20.4 10.91C18.9 10.91 17.29 11.44 17.29 12.6C17.29 12.68 17.33 16.2 17.33 16.2C15.98 16.63 15 17.37 15 18.42C15 18.56 15.09 37.38 15.09 37.38C15.17 38.74 17.43 39.76 20.34 39.76C23.25 39.76 25.51 38.74 25.59 37.38V37.36C25.59 37.36 25.67 18.56 25.67 18.42C25.67 17.47 24.71 16.76 23.51 16.32ZM23.51 14.46H24.15V15.07H23.51V14.46ZM20.41 12.16C21.42 12.16 22.05 12.44 22.23 12.6C22.04 12.76 21.41 13.04 20.41 13.04C19.41 13.04 18.77 12.76 18.58 12.6C18.77 12.44 19.4 12.16 20.41 12.16ZM22.22 17.18V17.22C22.05 17.38 21.38 17.65 20.4 17.65C19.42 17.65 18.76 17.38 18.58 17.22V13.99C19.12 14.19 19.77 14.29 20.4 14.29C21.03 14.29 21.68 14.19 22.22 13.99V17.18ZM24.36 36.89L24.33 37.31C24.31 37.59 23.03 38.51 20.33 38.51C17.63 38.51 16.35 37.59 16.33 37.28L16.31 36.99V20.22C17.34 20.78 18.84 21.07 20.34 21.07C21.81 21.07 23.33 20.77 24.37 20.2V36.89H24.36ZM20.33 19.82C17.88 19.82 16.25 18.98 16.25 18.43C16.25 18.15 16.66 17.81 17.37 17.53C17.38 17.55 17.38 17.58 17.39 17.6C17.62 18.45 19.01 18.9 20.4 18.9C21.79 18.9 23.19 18.45 23.42 17.59V17.58C24.04 17.85 24.41 18.16 24.41 18.42C24.42 18.98 22.79 19.82 20.33 19.82Z" fill="#332E54"/>
+  <path d="M28.68 17.96L28.23 18.39L30.16 20.42L30.61 19.99L31.07 19.56L29.13 17.53L28.68 17.96Z" fill="#332E54"/>
```

Comment:
you need to replace all `fill=#332E54` with `fill=currentColor`
this is for the custom color prop in <Icon /> component to take effect